content/rtp-replacement/progress.html

```diff
@@ -11,12 +11,16 @@ <h2>Progress</h2>
 		<li>4/4 game titles.</li>
 		<li>13/13 Panoramas.</li>
 		<li>1/1 battle weapons.</li>
+		<li>5/5 system layouts.</li>
+		<li>3/3 battlesystem layouts.</li>
+		<li>1/1 game over screens.</li>
 	</ul>
```

Comment:
Singular,. Same for Battle weapon above

Reply:
I expected this comment. 🍰 When I had it this way, it seemed wrong to me, as I do these things intuitive. When reading 1/1 as "One out of a set of one things", plural made more sense to me, but you likely do not do this in english at all. I also thought about dropping the /X altogether. As "Already done" implies we have them all, so not needed.